src/RoslynInsertionTool/RoslynInsertionTool/RoslynInsertionTool.RazzleBuild.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Roslyn.Insertion
+{
+    static partial class RoslynInsertionTool
+    {
+        private static async Task<bool> CanBuildPartitionAsync(string relativePathToPartition, CancellationToken cancellationToken)
+        {
```

Comment:
Do we need this file anymore? it mentions Razzle build and didn't thin kwe did that for insertion.

Reply:
@jaredpar I'm not sure. There are a fair number of options for RIT.exe, and no doubt some of them are outdated/broken/irrelevant now. However, I'd rather not block getting this into roslyn-tools on cleaning that up.

Reply:
Agree. This isn't blocking feedback. More curiousity.